app/views/api_docs/pages/release_notes.md

```diff
@@ -4,8 +4,12 @@
   [`personal_statement`](/api-docs/reference#applicationattributes-object)
   field to 11624 chars
 - Introduce `work_history_break_explanation` field to
-  [`work_experience`](/api-docs/reference#workexperiences-object).  This
+  [`work_experience`](/api-docs/reference#workexperiences-object). This
   contains the candidate’s explanation for any breaks in work history.
+- Introduce `missing_gcses_explanation` field to
```

Comment:
This needs to go into a section with a new date now. Not sure about the wrapping!

Reply:
Done, and I've unwrapped the lines 👍